guides/common/modules/proc_uploading-content-larger-than-2-mib.adoc

```diff
@@ -0,0 +1,169 @@
+[id="uploading-content-larger-than-2-mib"]
+= Uploading content larger than 2 MiB
```

Comment:
Are you sure that changing MB to MiB is correct when the original and the split below uses MB? I don't think it is, it's a different unit... Check with a SME?

Reply:
Reverted!